nixos/tests/fontconfig-default-fonts.nix

```diff
@@ -8,6 +8,7 @@ import ./make-test-python.nix ({ lib, ... }:
   ];
 
   machine = { config, pkgs, ... }: {
+    fonts.fontconfig.enable = true;
     fonts.enableDefaultFonts = true; # Background fonts
     fonts.fonts = with pkgs; [
```

Comment:
I wonder what happens when you use `fonts.fonts` or `enableDefaultFonts` but maybe `fonts.fontconfig.enable` isn't `true`? Would it be worth making sure that doesn't happen?

Reply:
I don't think so; most modules we currently have will ignore any config if `enable` is set to false. That said, `fonts.fontconfig.enable` isn't a sibling of `fonts.enableDefaultFonts`, so maybe we should indeed have `enableDefaultFonts` imply `enable` (either automatically, or by throwing an error, or not even forcing it, just producing a warning). Likewise for `fonts.fonts` being nonempty.

Reply:
@lheckemann Yeah, I think throwing an error would be okay.